resources/views/components/molecules/_m-collection-block--artwork.blade.php

```diff
@@ -0,0 +1,18 @@
+<{{ $tag ?? 'li' }} class="m-listing{{ (isset($variation) and $variation === 'o-pinboard__item') ? ' m-listing--variable-height' : '' }}{{ (isset($variation)) ? ' '.$variation : '' }}"{!! (isset($variation) and strrpos($variation, "--hero") > -1 and !$item->videoFront) ? ' data-behavior="blurMyBackground"' : '' !!} itemscope itemtype="http://schema.org/CreativeWork">
```

Comment:
This is a bit confusing. Could you use the `@php` directive to construct the class in a variable, and then use the variable in the html, for example:
```html
@php
    $class = isset($variation) ? ...
@endphp

<{{ $tag ?? 'li' }} class="{{ $class }}" itemscope itemtype="http://schema.org/CreativeWork">
```

Reply:
updated!